src/Aquarius.Client/Samples/Client/ServiceModel.cs

```diff
@@ -561,15 +612,59 @@ public class GetFieldVisitStatistics : IReturn<FieldVisitStatistics>
         public string Id { get; set; }
     }
 
+    [Route("/v1/filters", "GET")]
+    public class GetFilters : IReturn<SearchResultFilter>
+    {
+        
+    }
+
+    [Route("/v1/filters", "POST")]
+    public class PostFilter : IReturn<Filter>
+    {
+        public string Id { get; set; }
+        public string CustomId { get; set; }
+        public Instant? StartTime { get; set; }
+        public Instant? EndTime { get; set; }
+        public List<ObservedProperty> ObservedProperties { get; set; }
+        public List<SamplingLocation> SamplingLocations { get; set; }
+        public string Description { get; set; }
+        public AuditAttributes AuditAttributes { get; set; }
+    }
+
+    [Route("/v1/filters/{id}", "GET")]
+    public class GetFilter : IReturn<Filter>
+    {
+        public string Id { get; set; }
+    }
+
+    [Route("/v1/filters/{id}", "PUT")]
+    public class PutFilter : IReturn<Filter>
+    {
+        public string Id { get; set; }
+        public string CustomId { get; set; }
+        public Instant? StartTime { get; set; }
+        public Instant? EndTime { get; set; }
+        public List<ObservedProperty> ObservedProperties { get; set; }
+        public List<SamplingLocation> SamplingLocations { get; set; }
+        public string Description { get; set; }
+        public AuditAttributes AuditAttributes { get; set; }
+    }
+
+    [Route("/v1/filters/{id}", "DELETE")]
+    public class DeleteFilter : IReturnVoid
+    {
+        public string Id { get; set; }
+    }
+
     [Route("/v1/labanalysismethods", "GET")]
-    public class GetLabAnalysisMethods : IReturn<SearchResultLabAnalysisMethod>
+    public class GetLabanalysismethods : IReturn<SearchResultAnalysisMethod>
```

Comment:
Hmm. The PascalCase-iness got lost only on these 4 operations names. I wonder why? None of the other operations seemed to be lowercase now.